src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectSystem/SnapshotResolver.cs

```diff
@@ -110,7 +110,7 @@ await _projectManager
             return miscDocument;
         }
 
-        _logger.LogTrace($"{documentFilePath} not found in {_projectManager.GetProjects().SelectMany(p => p.DocumentFilePaths)}");
+        _logger.LogTrace($"{documentFilePath} not found in {string.Join(", ", _projectManager.GetProjects().SelectMany(p => p.DocumentFilePaths))}");
```

Comment:
For this, and the above, should we add specific `Append` overloads to our interpolated string handler and make this sort of thing generic?

Reply:
Not a bad idea. I was thinking along similar lines yesterday. I was also wondering if we should make it easy to add line breaks and indents for joins like these. This is going to be an insanely long single line string.

Reply:
I think this depends on how one looks at logs. I usually look at them in VS Code, and having things on a single line is easiest to scan, then when I find the thing im interested in, I break it up for readability.

But automatic handling of line breaks, such that subsequent lines were automatically indented, would probably be fine for scanning.